common/src/java/com/zimbra/common/soap/MailConstants.java

```diff
@@ -856,6 +857,10 @@ private MailConstants() {
     // messages
     public static final String E_MIMEPART = "mp";
     public static final String E_SUBJECT = "su";
+
+    public static final String E_OP = "op";
+    public static final String E_TN = "tn";
```

Comment:
can we be more descriptive with variable name

Reply:
Yes, edited and used more descriptive variable name